tools/building.py

```diff
@@ -1298,8 +1298,23 @@ def is_bitcode(filename):
 
 
 def is_wasm(filename):
-  magic = open(filename, 'rb').read(4)
-  return magic == b'\0asm'
+  if not os.path.exists(filename):
```

Comment:
could even be `os.path.isfile`?

Reply:
Interesting.. we tend to just use `exists` elsewhere but you are right we should probably use `isfile` more often.